Source/Scene/Expression.js

```diff
@@ -881,5 +925,182 @@ define([
+                return left + '[int(' + right + ')]';
+            case ExpressionNodeType.LITERAL_BOOLEAN:
+                return value ? 'true' : 'false';
+            case ExpressionNodeType.LITERAL_NUMBER:
+                return numberToString(value);
+            case ExpressionNodeType.LITERAL_STRING:
+                // The only supported strings are css color strings
+                color = Color.fromCssColorString(value, scratchColor);
+                if (defined(color)) {
+                    return colorToVec3(color);
+                }
+                return undefined;
+            case ExpressionNodeType.LITERAL_COLOR:
+                var args = left;
+                if (value === 'color') {
+                    if (!defined(args)) {
+                        return 'vec4(1.0)';
+                    } else if (args.length > 1) {
+                        var rgb = args[0];
+                        var alpha = args[1];
+                        if (alpha !== '1.0') {
+                            info.translucent = true;
+                        }
+                        return 'vec4(' + rgb + ', ' + alpha + ')';
+                    } else {
+                        return 'vec4(' + args[0] + ', 1.0)';
+                    }
+                } else if (value === 'rgb') {
+                    return 'vec4(' + args[0] + ' / 255.0, ' + args[1] + ' / 255.0, ' + args[2] + ' / 255.0, 1.0)';
+                } else if (value === 'rgba') {
+                    if (args[3] !== '1.0') {
+                        info.translucent = true;
+                    }
+                    return 'vec4(' + args[0] + ' / 255.0, ' + args[1] + ' / 255.0, ' + args[2] + ' / 255.0, ' + args[3] + ')';
+                } else if (value === 'hsl') {
+                    color = convertHSLToRGB(this);
+                    if (defined(color)) {
+                        return colorToVec4(color);
+                    } else {
+                        return 'vec4(czm_HSLToRGB(vec3(' + args[0] + ', ' + args[1] + ', ' + args[2] + ')), 1.0)';
+                    }
+                } else if (value === 'hsla') {
+                    color = convertHSLToRGB(this);
+                    if (defined(color)) {
+                        if (color.alpha !== 1.0) {
+                            info.translucent = true;
+                        }
+                        return colorToVec4(color);
+                    } else {
+                        if (args[3] !== '1.0') {
+                            info.translucent = true;
+                        }
+                        return 'vec4(czm_HSLToRGB(vec3(' + args[0] + ', ' + args[1] + ', ' + args[2] + ')), ' + args[3] + ')';
+                    }
+                }
+                break;
+            default:
+                // Not supported: FUNCTION_CALL, ARRAY, REGEX, VARIABLE_IN_STRING, LITERAL_NULL, LITERAL_REGEX, LITERAL_UNDEFINED
```

Comment:
Some of these can probably be explicitly disallowed by the spec (I suppose we can't get around this since any sane point cloud styling engine will use a shader) like `regex`, but please mark this in the task list so we can think through it before merging.